packages/app/src/game/module/board/overlays/quickChatBox.tsx

```diff
@@ -0,0 +1,25 @@
+import { AppState } from "../../../../store";
+import React from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { QuickChatOption } from "@creature-chess/models";
+
+const QuickChatBox: React.FunctionComponent<{ sendingPlayerId: string }> = ({ sendingPlayerId }) => {
+
+	const currentChat = useSelector<AppState, any>(state => state.game.quickChat.find(chat =>
```

Comment:
I'm happy to leave this for a separate PR in future, but it would be more performant if `quickChat` was an object indexed by playerId rather than an array

This is because `find` is `O(n)` (takes longer with bigger arrays), whereas looking up by key is `O(1)` (always the same cost regardless of size)

```ts
useSelector<AppState, any>(state => state.game.quickChat[sendingPlayerId])
```

Reply:
![image](https://user-images.githubusercontent.com/2037007/126872870-be6ec71f-a539-4dd2-aed1-8b94fe21af9d.png)